components/ProjectModal/styles.ts

```diff
@@ -79,3 +83,55 @@ export const closeButtonStyles: CSSProperties = {
 export const additionalInfoStyles: CSSProperties = {
   width: '315px',
 };
+
+// STYLED COMPONENTS
+export const AllKDMS = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: 300px;
+  height: 203px;
+  gap: 12px;
+`;
+
+export const KeyDevelopmentMilestone = styled.div<{ status: boolean }>`
+  padding-top: 8px;
+  padding-left: 15px;
+  width: 127px;
+  height: 30px;
+  flex-shrink: 0;
+  border-radius: 12px 4px 4px 4px;
+  border-top: ${props =>
+    props.status
+      ? '0.5px solid #4974E0'
+      : '0.5px solid rgba(46, 58, 89, 0.20)'};
+  border-left: ${props =>
+    props.status
+      ? '0.5px solid #4974E0'
+      : '0.5px solid rgba(46, 58, 89, 0.20)'};
+  background: ${props =>
+    props.status
+      ? 'linear-gradient(0deg, rgba(73, 116, 224, 0.00) 0%, rgba(73, 116, 224, 0.08) 100%)'
+      : 'none'};
```

Comment:
Given this complexity, it almost seems worthwhile to create a separate react component where this logic is handled more cleanly in js. Could also split this component down further.